xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,9 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!StringUtils::EndsWithNoCase(dllName, "libdvdread.dll"))
```

Comment:
AFAIK there are quite a few binary addons using this. IIRC game addons use the VFS paths to access zip:// paths and to be able to use ROMs over HTTP etc. Kodi core provides a lot of stuff through VFS paths that can't be interpreted otherwise.